validator/client/aggregate.go

```diff
@@ -129,6 +131,10 @@ func (v *validator) signSlotWithSelectionProof(ctx context.Context, pubKey [fiel
 	if err != nil {
 		return nil, err
 	}
+	fork, err := forks.Fork(slots.ToEpoch(slot))
```

Comment:
Do you still need this?

Reply:
nope must have missed in merge conflicts still in the process of cleaning that up after breaking up the prs.